src/index.js

```diff
@@ -1,17 +1,17 @@
 /* eslint-disable comma-dangle */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.